stories/format.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Style/Format" />
+
+# Format
+
+## Number format
+
+- Use of [space] for [internationally recommended](https://en.wikipedia.org/wiki/Decimal_separator) thousands separators. No comma or point, as these are reserved for use as the decimal sign.
+- Numbers whose magnitude is less than 1, the decimal sign should be preceded by a zero.
+
+## Date format
+
+### Code Description
+
+| Code | Description                                | Example |
+| ---- | ------------------------------------------ | ------- |
+| YYYY | Year on 4 digits                           | 2021    |
+| EEE  | Weekday on 3 letters, first with uppercase | "Mon"   |
+| DD   | Day of the month with leading zero         | 08      |
+| MM   | Numerical month with leading zero          | 07      |
+| MMM  | Month abbreviation on 3 letters uppercase  | "JUL"   |
+| HH   | Hour [0-23]                                | 13      |
+| mm   | Minute with leading zero                   | 05      |
+| ss   | Second with leading zero                   | 06      |
+
+### Format list
+
+| Alias   | code                     | Example                  | Length | Context                                              |
+| ------- | ------------------------ | ------------------------ | ------ | ---------------------------------------------------- |
+| Short#1 | DD MMM                   | 20 Jul                   | 5      | Chart time axis                                      |
+| Short#2 | DDMMM HH:mm              | 20JUL 09:00              | 11     | Limited space - Dense tables, charts                 |
+| Short#3 | YYYY-MM-DD               | 2020-07-20               | 10     |                                                      |
+| Mid#1   | EEE YYYY-MM-DD HH:mm     | Mon 2020-07-20 09:00     | 20     | Checking with agenda for day of the week             |
+| Mid#2   | YYYY-MM-DD HH:mm         | 2020-07-20 09:00         | 16     | No need of day of the week                           |
+| Mid#3   | YYYY-MM-DD HH:mm:ss      | 2020-07-20 09:00:00      | 19     | When the seconds are needed                          |
+| Full#1  | EEE MMM DD YYYY HH:mm:ss | Mon Jul 20 2020 09:00:00 | 24     | When a lot of space (hover) When precision is needed |
```

Comment:
Shouldn't this be internationalised and for example use `MM/DD/YYYY` in the US, and `DD/MM/YYYY` in Europe ?

Reply:
Yes, from the long-term point of view. For that, we would need to add more "clues" about whether we are displaying US format or non-US format.
Having an English language and just for example "12/11/2021" is not enough for knowing if it's December or November of 2021. So we might need to add more information (letters, like 12NOV2021, etc.).
To avoid ambiguity, the easier and most straight forward way would be to use the YYYY-MM-DD format for the moment.